File: 0283-move-zeroes/0283-move-zeroes.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        j=0
        c=0
        for i in range(len(nums)):
            if(nums[i]!=0):
                nums[j]=nums[i]
                j+=1
                c+=1
        for i in range(len(nums)-1,c-1,-1):
            nums[i]=0


        """
        Do not return anything, modify nums in-place instead.
        """
        

File: 0283-move-zeroes/0283-move-zeroes.cpp
class Solution {
public:
    void moveZeroes(vector<int>& nums) {
        int k=0;
        for(int i=0;i<nums.size();i++){
            if(nums[i]!=0){
                nums[k]=nums[i];
                k++;   
            }
        }
        for (int z=nums.size()-1;z>=k;z--){
            nums[z]=0;
        }
        
    }
};